lib/counter_culture/collection.rb

```diff
@@ -0,0 +1,72 @@
+module CounterCulture
+  class Collection
+    include Enumerable
+
+    def initialize(arr=Array.new)
+      @arr = arr
+    end
+
+    def fix_counts(options={})
```

Comment:
This logic is logically not related to the concept of the collection, but to the [`Reconciler`](https://github.com/magnusvk/counter_culture/blob/master/lib/counter_culture/reconciler.rb). It belongs there.